README.md

```diff
@@ -1,49 +1,51 @@
 # SuperTuxKart
 [![Build Status](https://travis-ci.org/supertuxkart/stk-code.svg?branch=master)](https://travis-ci.org/supertuxkart/stk-code)
-[![AppVeyor Build Satus](https://ci.appveyor.com/api/projects/status/github/supertuxkart/stk-code?svg=true&branch=master)](https://ci.appveyor.com/project/supertuxkart/stk-code)
-[![#supertuxkart on freenode](https://img.shields.io/badge/freenode-%23supertuxkart-brightgreen.svg)](https://webchat.freenode.net/?channels=supertuxkart)
+[![AppVeyor Build Status](https://ci.appveyor.com/api/projects/status/github/supertuxkart/stk-code?svg=true&branch=master)](https://ci.appveyor.com/project/supertuxkart/stk-code)
+[![#supertuxkart on the freenode IRC network](https://img.shields.io/badge/freenode-%23supertuxkart-brightgreen.svg)](https://webchat.freenode.net/?channels=supertuxkart)
 
 SuperTuxKart is a free kart racing game. It focuses on fun and not on realistic kart physics. Instructions can be found on the in-game help page.
 
 The SuperTuxKart homepage can be found at <https://supertuxkart.net/>. There is also our [FAQ](https://supertuxkart.net/FAQ) and information on how get in touch with the [community](https://supertuxkart.net/Community)
 
 ## Hardware Requirements
-* You need a 3D graphics card. (NVIDIA GeForce 8xxx and higher, ATI Radeon HD 4xxx and higher or Intel HD 3000 and higher.)
-* You should have a CPU that's running at 1GHz or better.
-* You'll need at least 512 MB of free VRAM (video memory).
-* Disk space: 400MB
-* Ideally, you want a joystick with at least 6 buttons.
+To run SuperTuxKart, make sure that your computer's specifications are equal or higher than the following specifications:
+
+* A graphics card capable of 3D rendering - NVIDIA GeForce 400 series and newer (GeForce GT 430 or newer), AMD/ATI Radeon HD 5000 series and newer, Intel HD Graphics 3000 and newer. OpenGL >= 3.3
```

Comment:
I have a GTX 295 that works great with STK. According to this:

http://nvidia.custhelp.com/app/answers/detail/a_id/2443/~/what-nvidia-products-support-opengl-3.0%2F3.1%3F

GeForce 8000 series card and newer support OpenGL 3.1.